akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -4,59 +4,33 @@
 
 package akka.remote.artery
 
-import java.nio.channels.FileChannel
-import java.nio.file.Path
 import java.util.concurrent.TimeUnit
-import java.util.concurrent.atomic.AtomicBoolean
-import java.util.concurrent.atomic.AtomicLong
-import java.util.concurrent.atomic.AtomicReference
+import java.util.concurrent.atomic.{ AtomicBoolean, AtomicLong, AtomicReference }
 
-import scala.annotation.tailrec
-import scala.concurrent.Await
-import scala.concurrent.Future
-import scala.concurrent.Promise
-import scala.concurrent.duration._
-import scala.util.Failure
-import scala.util.Success
-import scala.util.Try
-import scala.util.control.NoStackTrace
-import scala.util.control.NonFatal
-
-import akka.Done
-import akka.NotUsed
-import akka.actor.Actor
-import akka.actor.Props
-import akka.actor._
+import akka.{ Done, NotUsed }
+import akka.actor.{ Actor, Props, _ }
 import akka.annotation.InternalStableApi
 import akka.dispatch.Dispatchers
-import akka.event.Logging
-import akka.event.LoggingAdapter
-import akka.remote.AddressUidExtension
-import akka.remote.RemoteActorRef
-import akka.remote.RemoteActorRefProvider
-import akka.remote.RemoteTransport
-import akka.remote.UniqueAddress
+import akka.event.{ Logging, LoggingAdapter }
+import akka.remote._
 import akka.remote.artery.Decoder.InboundCompressionAccess
 import akka.remote.artery.Encoder.OutboundCompressionAccess
-import akka.remote.artery.InboundControlJunction.ControlMessageObserver
-import akka.remote.artery.InboundControlJunction.ControlMessageSubject
+import akka.remote.artery.InboundControlJunction.{ ControlMessageObserver, ControlMessageSubject }
 import akka.remote.artery.OutboundControlJunction.OutboundControlIngress
 import akka.remote.artery.compress.CompressionProtocol.CompressionMessage
 import akka.remote.artery.compress._
-import akka.remote.transport.ThrottlerTransportAdapter.Blackhole
-import akka.remote.transport.ThrottlerTransportAdapter.SetThrottle
-import akka.remote.transport.ThrottlerTransportAdapter.Unthrottled
-import akka.stream.AbruptTerminationException
-import akka.stream.ActorMaterializer
-import akka.stream.KillSwitches
-import akka.stream.Materializer
-import akka.stream.SharedKillSwitch
-import akka.stream.scaladsl.Flow
-import akka.stream.scaladsl.Keep
-import akka.stream.scaladsl.Sink
+import akka.remote.transport.ThrottlerTransportAdapter.{ Blackhole, SetThrottle, Unthrottled }
+import akka.stream._
+import akka.stream.scaladsl.{ Flow, Keep, Sink }
 import akka.util.{ unused, OptionVal, WildcardIndex }
 import com.github.ghik.silencer.silent
 
+import scala.annotation.tailrec
+import scala.concurrent.{ Await, Future, Promise }
```

Comment:
Automagic. Annoyingly enough it doesn't look like it is configurable in IntelliJ either, only how many imports to cause a wildcard import. Since we now have fail build on unused import I tend to often do an auto-fix-imports in IntelliJ because hand removal is tedious.

Reply:
Undid it now

Reply:
was it the grouping of imports? this is my settings:
<img width="870" alt="Screenshot 2019-09-04 at 15 37 08" src="https://user-images.githubusercontent.com/336161/64259789-03276400-cf2a-11e9-8a81-364f002d2e71.png">

Reply:
I didn't change any settings and a new auto-fix-imports made it good, but perhaps I did the previous time on a different computer with the _wrong settings(tm)_

Reply:
> because hand removal is tedious.

then we need to find a real solution for that, that works for everyone or turn off the warning again